src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -199,7 +199,11 @@ void ProblemManager::parseCommandLineInput()
     Path::pathPrefix() = splitPath( inputFileName ).first;
   }
 
-  if( opts.suppressMoveLogging )
+  if( opts.displayMoveLogging )
+  {
+    chai::ArrayManager::getInstance()->enableCallbacks();
```

Comment:
I could only find track of the callbacks here https://github.com/GEOSX/LvArray/blob/develop/src/ChaiBuffer.hpp#L447

So I guess it's find to do it like that right now, but we must be aware of this since it's a bit dangerous 😉

Reply:
The proposed change is intended to explicitly turn on/off callbacks using chai's methods ([here](https://github.com/LLNL/CHAI/blob/f90689b2958eda7025681c6390c7505a06509f6e/src/chai/ArrayManager.hpp#L389-L397)) instead of relying on chai ArrayManager's default constructor ([here](https://github.com/LLNL/CHAI/blob/f90689b2958eda7025681c6390c7505a06509f6e/src/chai/ArrayManager.cpp#L32)), which turn them on by default. Could you clarify why you believe this change could be potentially dangerous?

Reply:
Cleaning this up requires a slightly bigger refactoring. In particular, we need to remove the `setName("")` call from `ChaiBuffer` constructors, so that there's no logging by default and we aren't forced to disable callbacks globally. The method should also be renamed to reflect the purpose, e.g. `enableLogging(string id)`.

Then, on GEOSX side, in `Wrapper<>`, we can check if logging option is enabled in `GeosxState` and optionally call `enableLogging` on the wrapped object. Finally, we then remove all ad-hoc `setName()` calls on various objects (e.g. linear algebra matrices/vectors) and instead register them all in data repository, to manage this all centrally.

But this is a topic for another PR :)